apps/web-app/app/routes/import.tsx

```
@@ -0,0 +1,317 @@
+            <input
+              ref={fileInputRef}
+              type="file"
+              accept={importType === 'json' ? '.json' : '.csv'}
+              onChange={handleFileUpload}
+              disabled={importing}
+            />
+            <button
+              onClick={handleValidateOnly}
+              disabled={importing || !fileInputRef.current?.files?.[0]}
+              className="button secondary"
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)